docs/features/ValueTuples.cs

```diff
@@ -24,8 +24,89 @@ public interface ITuple
         object this[int i] { get; }
     }
 
-    public static class ValueTuple
+    public struct ValueTuple : IEquatable<ValueTuple>, IStructuralEquatable, IStructuralComparable, IComparable, ITupleInternal, ITuple
     {
+        public override Boolean Equals(Object obj)
```

Comment:
Let me hold on that. I will talk with corefx team this week and this code will go in their repo. 
Since it is based off `System.Tuple` which is their code, I'm assuming they will prefer consistency.